.travis.yml

```diff
@@ -10,23 +10,25 @@ cache:
     - $HOME/.composer/cache
 
 php:
+  - 7.4
```

Comment:
Need to address PHP 7.4 version and then this is good.

Reply:
Switched to "7.4snapshot", thanks for catching that!